x-pack/plugins/security_solution/common/api/detection_engine/rule_monitoring/model/execution_metrics.schema.yaml

```diff
@@ -25,3 +25,17 @@ components:
           description: Duration in seconds of execution gap
           type: integer
           minimum: 0
+        gap_range:
+          description: Range of the execution gap
+          type: object
+          required:
+            - from
+            - to
+          properties:
+            from:
+              description: Start of the execution gap
```

Comment:
Can we add clarification that this is a datestring? It's not obvious from the schema otherwise